src/scrapers/mizrahi.ts

```diff
@@ -70,9 +70,18 @@ function createLoginFields(credentials: ScraperSpecificCredentials) {
   ];
 }
 
+async function isLoggedIn(options: { page?: Page | undefined } | undefined) {
+  if (!options?.page) {
+    return false;
+  }
+  const oshXPath = `//a//span[contains(., "${checkingAccountTabHebrewName}") or contains(., "${checkingAccountTabEnglishName}")]`;
+  const oshTab = await options.page.$x(oshXPath);
+  return oshTab.length > 0;
+}
+
 function getPossibleLoginResults(page: Page): PossibleLoginResults {
   return {
-    [LoginResults.Success]: [AFTER_LOGIN_BASE_URL, async () => !!(await page.$x(`//a//span[contains(., "${checkingAccountTabHebrewName}") or contains(., "${checkingAccountTabEnglishName}")]`))],
+    [LoginResults.Success]: [AFTER_LOGIN_BASE_URL, isLoggedIn],
     [LoginResults.InvalidPassword]: [async () => !!(await page.$(invalidPasswordSelector))],
```

Comment:
Was always true because the return value of `page.$x` is an array, and `!!([])` is true.